src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -134,6 +135,22 @@ public boolean isIncludeBinaries() {
     }
 
     /**
+     * Sets the location to write a sample config file.
```

Comment:
I don't think we should include the `writeConfig` option in the config file.  The option triggers writing the file to disk, but I wouldn't want subsequent uses of the config file to overwrite itself (which is what including the option in the file itself would trigger).  If everything worked properly, then the config file wouldn't change.  If anything went amiss, it seems like you could mess up the config you were using.